bite-size-notes/week-8-e-Bayes Rule-RBE-RBF.md

```diff
@@ -0,0 +1,94 @@
+# Bayes Rule, Recursive Bayesian Estimator, and the Recursive Bayesian Filter
+
+## Bayes's Rule
+
+Bayes's rule, named after Reverend Thomas Bayes, is a fundamental principle in probability theory and statistics. It describes how to update the probability of a hypothesis based on new evidence. Mathematically, Bayes's rule is expressed as:
+$$P(X|Y) = \frac{P(Y|X)P(X)}{P(Y)}$$
+
+Each component of the expression is important and has its own name:
+- $P(X|Y)$ is the *posterior*, which asks "if y is a measurement, then what is x given y?"
+- $P(X)$ is the *prior*, a belief/guess of X before knowing Y
+- $P(Y|X)$ is the *likelihood function*, which is what you expect Y to be given X
+- $P(Y)$ (also written as $\int_X P(Y|X)P(X)dX$) is the normalizing factor.
+
+Yet what happens if we have multiple measurements? We can write our multiple measurement expression as the following:
+$$P(X|Y_1, Y_2) \propto P(Y_2|X,Y_1)P(X|Y_1)$$
```

Comment:
Explain why we're using proportionality instead of equality, and maybe mention proportional form of Bayes' rule in general

Reply:
Added a sentence highlighting that the \propto is used since Baye's rule needs a normalizaton step for the exact answer.